src/components/UI/Text/Textarea.vue

```diff
@@ -206,15 +208,14 @@ export default {
   methods: {
     //update HTML on input change in textarea
     updateChange() {
-      //invoked on input change
+      //invoked on input changeconsole.
```

Comment:
Be a little more descriptive
Write proper sentences
`this method is invoked the user types in the input box`

Also, re-verify for typos before sending the PR for review! There's a small typo here